eng/platform-matrix.yml

```diff
@@ -242,7 +242,8 @@ jobs:
         osGroup: Windows_NT
         osIdentifier: Windows_NT
         helixQueues:
-        # TODO: add Windows.10.Arm64.Open once Jenkins has been shutdown
+        - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')) }}:
```

Comment:
I think it would be helpful to have a comment here explaining why it's limited to CI (namely, capacity concerns).